### DISCUSSION TITLE
OBS Version

### DISCUSSION THREAD

**Discussion by larsonguy** [cpyarger/obs-midi#178] [Q&A] [answered]
Hi! Is there a minimum OBS version that this is compatible on? I recently had to downgrade to OBS 24.0.3 due to game capture issues and i re-ran the installer for the midi controller but I'm not seeing the tool show up under the Tools drop down. Any suggestions?

**Comment by cpyarger** [comment, marked as answer]
25.0 is when we started work on the plugin. upcoming versions will only be compatible with 27.2+ due to the new features added to OBS, testing is always done on the latest version available for download from the OBS website. I don't have the resources to code or test for backwards compatibility unfortunately. 

_insert gripe about lack of people interested in pitching into code work_ LOL
